<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>491</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>491</width>
      <height>121</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>121</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0b">
         <property name="text">
          <string>0</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">born</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_1b">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">born</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_2b">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">born</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_3b">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>3</string>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">born</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_4b">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">born</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_5b">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">born</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_6b">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>6</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">born</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_7b">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>7</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">born</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_8b">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">born</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0s">
         <property name="text">
          <string>0</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">survival</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_1s">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">survival</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_2s">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">survival</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_3s">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>3</string>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">survival</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_4s">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">survival</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_5s">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">survival</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_6s">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>6</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">survival</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_7s">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>7</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">survival</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_8s">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">survival</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="speed">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="step_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mix_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Mix</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="clean_button">
         <property name="text">
          <string>Clean</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lrb">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Load rule</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="srb">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Saves of rules</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lfb">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Load figure</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sfb">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Saves of figures</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>505</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="survival">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="born">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
